src/main/java/org/minions/devfund/yuri/battleship/Ocean.java

```diff
@@ -0,0 +1,56 @@
+package org.minions.devfund.yuri.battleship;
+
+
+import java.util.stream.IntStream;
+
+/**
+ * Represents the ocean in the game.
+ */
+public class Ocean {
+    public static final int SHIPS_AMOUNT = 20;
+    public static final int SHIPS_SUNK = 13;
+    private Ship[][] ships;
+    private int shotsFired;
+    private int hitCount;
+    private int shipsSunk;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "shipsSunk" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)